source/NVDAObjects/IAccessible/winword.py

```diff
@@ -234,8 +246,17 @@ def script_setColumnHeader(self,gesture):
 			else:
 				# Translators: a message reported in the SetColumnHeader script for Microsoft Word.
 				ui.message(_("Cannot find row {rowNumber} column {columnNumber}  in column headers").format(rowNumber=cell.rowIndex,columnNumber=cell.columnIndex))
-	script_setColumnHeader.__doc__=_("Pressing once will set this cell as the first column header for any cells lower and to the right of it within this table. Pressing twice will forget the current column header for this cell.")
 
+	@script(
+		gesture="kb:NVDA+shift+r",
+		# Translators: The label of a shortcut of NVDA.
+		description=_(

```

Comment:
my understanding is that descriptions should be short like "set row header", and detailed descriptions should be in the user guide.

At the very least, the first sentence should be a short description

```suggestion
		description=_(
			"Set row header."
```

Reply:
Hi, my intention for this suggestion was to add a shorter description before the longer description, not to remove it entirely.
If you want to remove this longer description please ensure there it is included in the user guide.

Reply:
ah I see it is.